Src/EmbeddedScripts.Lua.Moonsharp/MoonsharpRunner.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Threading.Tasks;
+using EmbeddedScripts.Shared;
+using MoonSharp.Interpreter;
+
+namespace EmbeddedScripts.Lua.Moonsharp
+{
+    public class MoonsharpRunner : ICodeRunner
+    {
+        private Script _script = new();
+        
+        public Task RunAsync(string code)
+        {
+            _script.DoString(code);
+            
+            return Task.CompletedTask;
+        }
+
+        public ICodeRunner Register<T>(T obj, string alias)
+        {
+            if (obj is not Delegate)
```

Comment:
Do you need to register type if obj type is primitive type (for example, int)?

Reply:
No. Type registration is required for custom types which are userdata in Lua [https://www.moonsharp.org/mapping.html](more on types conversion). May be there will be also required to identify dictionaries and lists as well as I identify delegates. I will add this test cases, and make changes if needed.